File: mavit-api/src/main/java/uno/csci4830/mavitapi/payload/response/HomePageResponse.java
package uno.csci4830.mavitapi.payload.response;

import lombok.Getter;
import lombok.Setter;
import uno.csci4830.mavitapi.model.College;
import uno.csci4830.mavitapi.model.Major;
import uno.csci4830.mavitapi.model.University;

import java.util.Set;

@Getter
@Setter
public class HomePageResponse {

    private Set<University> universities;
    private Set<College> colleges;
    private Set<Major> majors;

}

File: mavit-client/src/app/payloads/responses/HomePageResponse.ts
import { College } from 'src/app/models/college'
import { Major } from 'src/app/models/major'
import { University } from 'src/app/models/university'

export class HomePageResponse {
    
    universities : University[] = []
    colleges : College[] = []
    majors : Major[] = []
}